src/screens/ProfileScreen/index.js

```diff
@@ -167,7 +168,7 @@ const ProfileScreen = ({route}) => {
   const [opacity, setOpacity] = React.useState(0);
   const [tempBio, setTempBio] = React.useState('');
   const [tempFullName, setTempFullName] = React.useState('');
-  const [isLoadingUpdateImageGalery, setIsLoadingUpdateImageGalery] = React.useState(false);
+  const [isLoadingUpdateImageGallery, setIsLoadingUpdateImageGallery] = React.useState(false);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This change corrects a typo in the variable name, improving readability and maintainability. Good job!
```diff
- const [isLoadingUpdateImageGalery, setIsLoadingUpdateImageGalery] = React.useState(false);
+ const [isLoadingUpdateImageGallery, setIsLoadingUpdateImageGallery] = React.useState(false);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->